sdk/java-sdk-protobuf/src/main/java/kalix/javasdk/HttpResponse.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2024 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package kalix.javasdk;
+
+
+import javax.annotation.Nullable;
+import java.util.Optional;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+public class HttpResponse {
+  private final int statusCode;
+  private final Optional<String> contentType;
+  private final byte[] body;
```

Comment:
@octonato I know that you suggested implementing the body as `Object`, but I'm not sure if this is a good idea, because of the deserialization. What should be under the body in that case? With an array of bytes, we are crystal clear, put your bytes here, and you will get them back. It's on you to ser/der them correctly. If you want auto-serarialization, then you are simply using low-level abstraction, go with Kalix default behavior.

Reply:
Ok, check now.

Reply:
Thanks. I like the way you make it now.